plugins/catalog-backend/src/database/Database.ts

```diff
@@ -56,6 +57,8 @@ function entityRequestToDb(request: DbEntityRequest): DbEntitiesRow {
   return {
     id: '',
     location_id: request.locationId || null,
+    etag: uuidv4(), // TODO(freben): Atomicity isn't checked using these yet
```

Comment:
Maybe we could make this a random base62 string? just to not confuse it with an ID

Reply:
Dude don't even, haha, base62 is such a wonky thing. I can base64 encode it tho, sure

Reply:
base62 is nicer here imo, because we never want to decode it, just generate some random string. base64 is gonna give you some of `-/_+`

Reply:
I mean you can really just `base64(random()).replace(/[a-z0-9]/i, '')`

Reply:
Mm i wasn't worried about the other chars for that exact reason, it's just an opaque string that you never deal with in any other way than as a blob. But ok I can make it alphanum pure